src/engraving/libmscore/symbol.cpp

```diff
@@ -165,20 +148,6 @@ String FSymbol::accessibleInfo() const
     return toString();
 }
 
-//---------------------------------------------------------
-//   draw
-//---------------------------------------------------------
-
-void FSymbol::draw(mu::draw::Painter* painter) const
```

Comment:
It looks like FSymbol is not yet handled in TDraw/SingleDraw. (Or I might be missing something)

Reply:
hmm, it was missed, I added it